config/mongoid.yml

```diff
@@ -34,7 +34,7 @@ production:
       uri: <%= ENV['MONGODB_URI'] %>
       options:
         write:
-          w: 1
+          w: 3 # prevents race conditions
```

Comment:
Will this prevent local development?

Reply:
Good question. I've just tried `govuk-docker-up` and can run the app locally and exercise the "Add new service" functionality, which seems to persist. So it does appear to allow local development.